.agents/skills/git-worktree/scripts/worktree-manager.sh

```
@@ -0,0 +1,500 @@
+  cd "$worktree_path"
+  echo -e "${BLUE}Now in: $(pwd)${NC}"
+}
+
+# Copy env files to an existing worktree (or current directory if in a worktree)
+copy_env_to_worktree() {
+  local worktree_name="$1"
+  local worktree_path
+
+  if [[ -z "$worktree_name" ]]; then
+    # Check if we're currently in a worktree
+    local current_dir=$(pwd)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)